app/controllers/CheckEmploymentExpensesController.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import common.SessionValues.EXPENSES_CYA
+import config.AppConfig
+import controllers.predicates.AuthorisedAction
+import models.GetEmploymentExpensesModel
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+import utils.SessionHelper
+import views.html.expenses.CheckEmploymentExpensesView
+
+import javax.inject.Inject
+
+class CheckEmploymentExpensesController @Inject()(authorisedAction: AuthorisedAction,
```

Comment:
can move this controller into employment folder in controllers package

Reply:
Done